.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: "Docker"
+
+# we're running this on pull requests so we can test it
+on:
+  pull_request:
+# FIXME: we want this to create a "latest" release on changes
+# to `dev`, and actual releases when we create "release" branches;
+# does this do that?
+# on:
+#   push:
+#     branches: ['dev', 'release']
```

Comment:
Two recommendations for releasing the system docker image:
- On pushes to `dev` with a `latest` or `stable` tag
- On pushes to release branches with the release branch name as the docker tag
These two methods for pushing images make sense assuming that pushing to `dev` or release branches only occurs when pull requests are merged into those branches.